proxy/airdropper/airdropper.py

```diff
@@ -379,5 +501,19 @@ def process_receipts(self):
             self.current_slot = meta.block_slot
             if meta.tx['transaction']['message']['instructions'] is not None:
                 self.process_trx_airdropper_mode(meta.tx)
+                self.process_trx_neon_instructions(meta.tx)
         self.latest_processed_slot = self._sol_tx_collector.last_block_slot
+
+        # Find the minimum start_block_slot through unfinished neon_large_tx. It is necessary for correct
+        # transaction processing after restart the airdrop service. See `process_trx_neon_instructions`
+        # for more information.
+        outdated_holders = [tx.key for tx in self.neon_large_tx.values() if tx.last_block_slot + self._config.holder_timeout < self._sol_tx_collector.last_block_slot]
+        for tx_key in outdated_holders:
+            LOG.info(f"Outdated holder {tx_key}. Drop it.")
+            self.neon_large_tx.pop(tx_key)
```

Comment:
Here is `tx_key`, because `NeonIndexedHolderInfo.key` already takes `value` (https://github.com/neonlabsorg/proxy-model.py/blob/d7fc919348c11ac7a40c00a1414a594f969d6ae8/proxy/indexer/indexed_objects.py#L116)